src/main/java/pinger/Pinger.java

```diff
@@ -8,25 +8,32 @@
 import java.net.URL;
 
 public class Pinger {
-    public static void main(String[] args) throws IOException {
-        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
-            System.out.println("Enter the URL to be pinged");
-            String urlI = "http://" + reader.readLine();
-            URL url = new URL(urlI);
-            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
-            try (InputStream stream = conn.getInputStream()) {
-                String content = readAllLines(stream);
-                if (content.isEmpty()) {
-                    content = "No content returned";
+    public static void main(String[] args)  {
+        char ch=y;
```

Comment:
are you sure this compiles?

Reply:
its just a declaration of char type and do while loop, so there should be no errors